pkg/reporter/reporter.go

```diff
@@ -61,13 +66,69 @@ func Post(report inventory.Report, anchoreDetails config.AnchoreInfo) error {
 	return nil
 }
 
+type anchoreVersion struct {
+	API struct {
+		Version string `json:"version"`
+	} `json:"api"`
+}
+
+// This method retrieves the API version from Anchore
+//
+//nolint:gosec
+func getVersion(anchoreDetails config.AnchoreInfo) (int, error) {
+	log.Debug("Detecting Anchore API version")
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: anchoreDetails.HTTP.Insecure},
+	}
+	client := &http.Client{
+		Transport: tr,
+		Timeout:   time.Duration(anchoreDetails.HTTP.TimeoutSeconds) * time.Second,
```

Comment:
This doesn't support passing additional headers; however, it appears that k8s-inventory in general doesn't support this yet, so we'll need to add that in future

Reply:
What additional headers were you thinking about?

Reply:
Well for anchorectl the primary ones that matter are the cloudflare ones

Reply:
So it has the config section like:
```yaml
http:
  headers:
    insecure: true
    CF-Access-Client-Id: "*****"
    CF-Access-Client-Secret: "*****"
```

Reply:
But the config is just a map of string:string so it can accept anything

Reply:
So it just looks like
```go
type HTTP struct {
	TLSInsecure bool              `yaml:"tls-insecure" json:"tls-insecure"  mapstructure:"tls-insecure" description:"disable SSL certificate verification for all HTTP calls (not recommended)"`
	Timeout     int               `yaml:"timeout" json:"timeout"  mapstructure:"timeout" description:"time in seconds before cancelling an HTTP request"`
	Headers     map[string]string `yaml:"headers" json:"headers" mapstructure:"headers" description:"default HTTP headers to add to all HTTP requests"`
}
```

Reply:
I followed the existing approach in `Post` which doesn't pass along additional headers from the config.
Should we address this in a separate PR?